src/client/content_cao.h

```diff
@@ -263,4 +264,12 @@ class GenericCAO : public ClientActiveObject
 	{
 		return m_prop.infotext;
 	}
+
+	bool doShowSelectionBox() override
```

Comment:
`content_cao.h` is included 6x in a source file, no header files.
For incremental builds this means that 6 files will need to be recompiled whenever this or one of the included header files change. In the worst case, there's a chain of header inclusions (`log.h` for example) - resulting in many, many source files that need to be recompiled even though there are no changes for them.

Hence, please move the definition to `content_cao.cpp` and keep the header light as possible.